x-pack/plugins/infra/public/observability_logs/log_stream_page/state/src/selectors.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useSelector } from '@xstate/react';
+import { LogStreamQueryActorRef } from '../../../log_stream_query_state';
+import { MatchedStateFromActor } from '../../../xstate_helpers';
+import { LogStreamPageActorRef } from './state_machine';
+
+export const useLogStreamQueryChildService = (
+  logStreamPageStateService: LogStreamPageActorRef
+): LogStreamQueryActorRef =>
+  useSelector(logStreamPageStateService, (state) => state.children.logStreamQuery);
```

Comment:
**nit**: could you reuse here the reference to the `selectLogStreamQueryChildService` function? or are these selector handlers duplicated on purpose?

Reply:
Good catch. This hook is actually a left-over that is not used anymore, so I'll remove it.